fireapp/lib/presentation/constraint_form/temporal_input_field.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:fireapp/presentation/constraint_form/base_input_field.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+abstract class TemporalInputField extends BaseInputField {
+  const TemporalInputField(
+      {super.key,
+      required super.controller,
+      required super.label,
+      required super.icon});
+
+  TextFormField buildContext(BuildContext context, void Function() onTap) {
+    return TextFormField(
+        controller: controller,
+        style: Theme.of(context).textTheme.labelLarge,
+        // using the same text theme from login_page
+        decoration: InputDecoration(
+          labelText: AppLocalizations.of(context)?.enterDate,
+          fillColor: Colors.white,
```

Comment:
You should be accessing this kind of information through Theme. If we wanted to add a dark mode, or a high contrast mode, these constants would prevent us from doing so.